spec/pg_eventstore/queries/subscription_queries_spec.rb

```diff
@@ -0,0 +1,276 @@
+# frozen_string_literal: true
+
+RSpec.describe PgEventstore::SubscriptionQueries do
+  let(:instance) { described_class.new(PgEventstore.connection) }
+
+  describe '#find_by' do
+    subject { instance.find_by(attrs) }
+
+    let(:attrs) { { set: 'Foo', name: 'Bar' } }
+
+    describe 'when subscription exists' do
+      let!(:subscription) { SubscriptionsHelper.create(**attrs) }
```

Comment:
Why don't you use FactoryBot?

Reply:
FactoryBot works with ActiveRecord. It will be hard to create the same interface as ActiveRecord has